src/java/org/apache/cassandra/config/Config.java

```diff
@@ -282,6 +282,13 @@
     public volatile int concurrent_materialized_view_builders = 1;
     public volatile int reject_repair_compaction_threshold = Integer.MAX_VALUE;
 
+    /**
```

Comment:
Modifications to this class are to fix breaking changes detected while diffing config cross 3.0, 3.11, and 4.0 releases